site/ja/blog/origin-trials/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Chromeのオリジントライアル入門
+subhead: オリジントライアルは、新規の、または実験的なウェブプラットフォームの機能をテストするための方法です。また、その機能が全てのユーザ向けに利用可能になる前に、その機能の使いやすさ、実用性、そして有効性についてウェブ標準コミュニティにフィードバックする方法でもあります。
+authors:
+  - samdutton
+date: 2020-06-22
+updated: 2021-09-01
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/KeaVCdXHWzrI35QRvsZL.jpg
+alt: Pipette with purple liquid
+tags:
+  - origin-trials
+---
+
+*翻訳者の [yoichiro](https://github.com/yoichiro) さんに感謝いたします。*
+
+<!-- Origin trials give you access to a new or experimental feature, to build
+functionality your users can try out for a limited time before the feature
+is made available to everyone. -->
+オリジントライアルにより、新規の、または実験的な機能へのアクセスと、機能が誰でも利用可能になる前に、限定された期間ユーザが試すことのできる機能の構築ができるようになります。
+
+<!-- When Chrome offers an origin trial for a feature, you can register for the trial to enable
+the feature for all users on your [origin](https://web.dev/same-site-same-origin/#origin),
+without requiring them to toggle any flags or switch to an alternative build
+of Chrome (though they may need to upgrade). Origin trials enable developers
+to build demos and prototypes using new features. The trials also help Chrome engineers
+understand how new features are used, and how they may interact with other web technologies. -->
+Chrome がある機能のオリジントライアルを提供している時は、あなたの [オリジン](https://web.dev/same-site-same-origin/#origin) 上で全てのユーザにその機能を有効にすするためにトライアルに登録することができます。そのために、Chrome のフラグを切り替えたり代替ビルドに切り替える必要はありません（ただし、アップグレードが必要な場合があります）。オリジントライアルにより、開発者は新しい機能を使用してデモやプロトタイプを作成できます。このトライアルは、Chrome エンジニアが新機能がどのように利用され、そして他のウェブテクノロジとどのように相互に利用されるかを理解するのにも役立ちます。
+
+<!-- Origin trials are public and open to all developers. They are limited in duration and
+usage. Participation is a self-managed process with limited documentation and support.
+Participants should be willing and able to work relatively independently using the
+documentation available, which, at this stage, will likely be limited to API
+specifications and explainers, though web.dev tries to provide guidance whenever
+possible. -->
+オリジントライアルは、全ての開発者に対して、パブリックであり、オープンです。それらは、期間と使用方法に制限があります。参加は、限られたドキュメントとサポートが準備された自己管理プロセスです。参加者は、利用可能なドキュメントを使用して、比較的独立して作業する意思と能力を備えている必要があります。この段階では、API 仕様とその説明者に限定される可能性がありますが、web.dev は可能な限りガイダンスを提供しようとします。
```

Comment:
```suggestion
オリジントライアルは、全ての開発者に対して、パブリックであり、オープンです。それらは、期間と使用方法に制限があります。参加は、限られたドキュメントとサポートが準備された自己管理プロセスです。参加者は、利用可能なドキュメントを使用して、比較的独立して作業する意思と能力を備えている必要があります。この段階では、API 仕様とその Explainer に限定される可能性がありますが、web.dev が可能な限りのガイダンスを提供します。
```
"Explainer" is a document prepared before writing a spec - example: https://github.com/samuelgoto/email-otp

Reply:
@agektmr I didn't know the "Explainer". Thank you!